hca/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.0.0'
+__version__ = '4.8.0'
```

Comment:
Looks like this accidentally got in.  Should change this back to `0.0.0`.

Reply:
! how did that get there, reverting, thanks!